frontend/sw360-portlet/src/main/resources/META-INF/resources/html/projects/includes/linkedProjects.jspf

```diff
@@ -66,25 +66,62 @@
             table.treetable("loadBranch", node, rows);
         });
 
-        $('#downloadLicenseInfoButton').click(downloadLicenseInfo);
-        $('#downloadSourceCodeBundleButton').click(downloadSourceCodeBundleButton);
-
         $('#btnExportGroup a.dropdown-item').on('click', function(event) {
             exportSpreadsheet($(event.currentTarget).data('type'));
         });
 
-        function downloadLicenseInfo() {
-            window.location.href = '<portlet:renderURL> \
-              <portlet:param name="<%=PortalConstants.PROJECT_ID%>" value="${project.id}"/> \
-              <portlet:param name="<%=PortalConstants.PAGENAME%>" value="<%=PortalConstants.PAGENAME_LICENSE_INFO%>"/> \
-            </portlet:renderURL>';
+        $('#downloadLicenseInfo a.dropdown-item').on('click', function(event) {
+            var type=$(event.currentTarget).data('type');
+            downloadLicenseInfo(type);
+        });
+
+        $('#downloadSourceCode a.dropdown-item').on('click', function(event) {
+            var type=$(event.currentTarget).data('type');
+            downloadSourceCodeBundleButton(type);
+        });
+
+        function downloadLicenseInfo(type) {
+            var licenseInfoChoice = type === 'projectWithSubProject' ? true : false;
+            var url = null;
+            if(licenseInfoChoice)
+            {
+                url = '<portlet:renderURL> \
+                <portlet:param name="<%=PortalConstants.PROJECT_ID%>" value="${project.id}"/> \
+                <portlet:param name="<%=PortalConstants.PAGENAME%>" value="<%=PortalConstants.PAGENAME_LICENSE_INFO%>"/> \
+                <portlet:param name="<%=PortalConstants.PROJECT_WITH_SUBPROJECT%>" value="true"/> \
```

Comment:
Is it possible to append the `licenseInfoChoice` directly to the `<portlet:param name="<%=PortalConstants.PROJECT_WITH_SUBPROJECT%>" value="true"/>`, instead of unnecessary if else condition.

Reply:
Changes implemented.